### PR TITLE
Update Cloud intro to new wording

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Update the Cloud intro to the new wording

# Links

- Partial: https://github.com/timescale/docs-private/issues/121
- Built off: https://github.com/timescale/docs/pull/2355

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: _partials/_cloud-data-tiering-beta.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ To request access to the private beta for data tiering:
 
 1.  Sign in to your [Timescale
     account](https://console.cloud.timescale.com/).
-2.  Click on the service where you want to enable data tiering.
-3.  Navigate to `Operations`, then `Data tiering`.
-4.  Click on `Request access` to send an email to the support team, who can
+1.  Click on the service where you want to enable data tiering.
```
Comment (Rajakavitha1) [inline _partials/_cloud-data-tiering-beta.md:6]:
```suggestion
1.  Click the service where you want to enable data tiering.
```

File: _partials/_cloud-data-tiering-beta.md [hunk 2]
```diff
@@ -3,9 +3,9 @@ To request access to the private beta for data tiering:
...
+1.  Navigate to `Operations`, then `Data tiering`.
+1.  Click on `Request access` to send an email to the support team, who can
```
Comment (Rajakavitha1) [inline _partials/_cloud-data-tiering-beta.md:8]:
```suggestion
1.  Click `Request access` to send an email to the support team, who can
```

File: _partials/_cloud-installation.md [hunk 3]
```diff
@@ -5,9 +5,9 @@
 1.  Sign up for a [Timescale account][sign-up] with your
     name and email address. You do not need to provide payment details to
     get started. A confirmation email is sent to the email address you provide.
-2.  Verify your email by clicking on the link in the email you received. Don't
+1.  Verify your email by clicking on the link in the email you received. Don't
```
Comment (Rajakavitha1) [inline _partials/_cloud-installation.md:8]:
```suggestion
1.  Verify your email by clicking the link in the email you received. Don't
```

File: _partials/_cloud-intro.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
-Timescale is hosted high-performance PostgreSQL for time-series and
-analytics. It gives you the reliability of PostgreSQL, the time-series
-superpowers of Timescale, and the peace of mind of a fully managed service
-with automatic backup and restore, high availability with replication, seamless
-scaling and resizing, and much more.
+Timescale is a Postgresql data platform for time-series, events and analytics.
```
Comment (Rajakavitha1) [inline _partials/_cloud-intro.md:1]:
```suggestion
Timescale is a PostgreSQL data platform for time-series, events and analytics.
```

File: _partials/_cloud-mst-comparison.md [hunk 5]
```diff
@@ -2,12 +2,11 @@
   type="cloud"
   header="Timescale or Managed Service for TimescaleDB?"
 >
-If your workload lives in AWS, use
-[Timescale](/getting-started/latest/),
-built and exclusively operated by Timescale, designed to offer maximum
-cost-effectiveness and performance for time-series data. If you need Azure or
-GCP, contact
+If your workload lives in AWS, use [Timescale](/getting-started/latest/). Built
+and exclusively operated by Timescale, Timescale is a Postgresql data platform
```
Comment (Rajakavitha1) [inline _partials/_cloud-mst-comparison.md:6]:
```suggestion
and exclusively operated by Timescale, Timescale is a PostgreSQL data platform
```

**Review by Rajakavitha1 — APPROVED** [review]

File: _partials/_cloud-mst-comparison.md [hunk 6]
```diff
@@ -2,12 +2,11 @@
...
-If your workload lives in AWS, use
-[Timescale](/getting-started/latest/),
-built and exclusively operated by Timescale, designed to offer maximum
-cost-effectiveness and performance for time-series data. If you need Azure or
-GCP, contact
+If your workload lives in AWS, use [Timescale](/getting-started/latest/). Built
+and exclusively operated by Timescale, Timescale is a PostgreSQL data platform
+for time-series, events and analytics.
+
+If you need Azure or GCP, contact
```
Comment (gina-alioto) [inline _partials/_cloud-mst-comparison.md:9]:
```suggestion
If you need Azure or GCP, contact
```
Do we need this box?  Can we remove it entirely?  If we can't remove it, can we simply remove from "If you need Azure or GCP..."

**Review by gina-alioto — COMMENTED** [review]
In the comparison box, perhaps we eliminate all reference to Azure and GCP?  We do not want individuals contacting Support for accounts.

File: _partials/_cloud-mst-comparison.md (hunk 6 above)
Comment (Loquacity) [reply _partials/_cloud-mst-comparison.md:9]: So the point of this partial is to compare the two. We use it mostly on MST pages, to try and direct people back to Cloud. For example: https://docs.timescale.com/mst/latest/

So, with that in mind, I don't think we can avoid talking about MST directly, which is why I removed the sign up link and redirected people to support. That way, if they're desperate enough for MST, we can get them to talk to someone directly, and qualify whether or not they actually need it.

**Pull request merged**